backend/metasfresh-dist/metasfresh-webui-api-remote-via-ssh-tunnel.launch

```diff
@@ -12,7 +12,7 @@
 <booleanAttribute key="org.eclipse.jdt.launching.ALLOW_TERMINATE" value="false"/>
 <mapAttribute key="org.eclipse.jdt.launching.CONNECT_MAP">
 <mapEntry key="hostname" value="localhost"/>
-<mapEntry key="port" value="18789"/>
+<mapEntry key="port" value="8789"/>
```

Comment:
i recommend to stick with 18789 and rather adopt your ssh config...but eiother way, ok with me